chapter1/scenarios/11b_Ghosts.cfg

```diff
@@ -131,7 +131,7 @@
     [/filter_condition]
     [message]
       speaker=Haldar
-      message=_"I told you to recruit ghosts not to recall! Try it again!"
+      message=_"I told you to recruit ghosts not to recall them! Try it again!"
```

Comment:
He meant that she should not recall any unit, not just ghost. And besides, she was not able to recruit ghosts in previous scenarios anyway.